workflow/rules/bcbio_variation_recall.smk

```diff
@@ -17,6 +17,9 @@ rule bcbio_variation_recall_ensemble:
         ref=config["reference"]["fasta"],
     output:
         vcf=temp("snv_indels/bcbio_variation_recall_ensemble/{sample}_{type}.ensembled.vcf.gz"),
+        bcbio_work=temp(
+            "snv_indels/bcbio_variation_recall_ensemble/{sample}_{type}.ensembled-work/{sample}_{type}.ensembled-intersect.txt"
```

Comment:
Why not making the whole dir temporary?

Reply:
I am not completely sure what the output looks like but if the directory only contains files which are temporary, I would do the `directory()` around the `"snv_indels/bcbio_variation_recall_ensemble/{sample}_{type}.ensembled-work"` and make the whole thing temporary. If it is only one file you want to get rid of then this is fine.